pypiserver/__main__.py

```diff
@@ -249,6 +257,19 @@ def main(argv=None):
             c.log_err_frmt = v
         elif k == "--cache-control":
             c.cache_control = v
+        elif k == "--auther":
+            try:
+                mod, _, func = v.rpartition(".")
+                if mod:
+                    c.auther = getattr(importlib.import_module(mod), func)
```

Comment:
Using importlib is an indicator that you need an extension mechanism; *setuptools8 provide an excellent such mechanism through the `entry_points`.   
I prefer to leav this commit to wait until we introduce "plugins."